go-mode.el

```diff
@@ -1474,28 +1474,33 @@ visit FILENAME and go to line LINE and column COLUMN."
       (if (buffer-modified-p)
           (message "Buffer is modified, file position might not have been correct")))))
 
-(defun godef--call (point)
+(defun godef--call (point with-type-info)
```

Comment:
I think this change would break external packages depending on (godef--call) as it changes the method signature. A quick search of github shows there are external callers (including company-go).

We probably could do this without breaking other users by using an optional flag, something like `(defun godef--call (point &optional only-location)`.

Reply:
Thanks for taking a look. Makes sense but I'm no longer using this, so no interest in revisiting the PR at this time